MySolution.rb

```diff
@@ -0,0 +1,36 @@
+class MySolution
+
+    #problem1
+    def sumOfEven(array)
+        array.select{|v| v.even?}.sum
+    end
+    
+    #problem2
+    def getLargest2Numbers(array)
+        bigMax = -1/0.0
+        smallMax = -1/0.0
+        for i in array
```

Comment:
Ruby code should be more expressive. 
Confusion here is that usually, when you do a `for`, the `i` is the index. In this case, it's the actual item in the array.